manifest.json

```diff
@@ -29,6 +29,15 @@
 					"en": "Choose a domain for Synapse",
 					"fr": "Choisissez un domaine pour Synapse"
 				},
+				"example": "synapse.domain.org"
+			},
+			{
+				"name": "server_name",
+				"type": "string",
+				"ask": {
+					"en": "If your synapse domain is a subdomain, you can choose a name for your Synapse server to have your matrix user-ids looking like @user:domain.org instead of @user:synapse.domain.org",
```

Comment:
Maybe we could add the possibility to have the same server_name than the domain by default. I think that is a specific use case, so I purpose to do add a default value with some thing like "Same than the domain". By this the user which don't really know what is this can just leave the default value.

Reply:
How do you propose to do that ?
We need to fix a default value, but that depends on each install. I don't know how to get the main domain, from inside the json :/

Reply:
I purpose one solution (but it's like hackish... because I don't think that we have actually the possibility to have a dynamic default value).

You add a default key as same as in line 50, and the value could be something like "Same than the domain".

In the install script you can add a code like that:

```bash
if [ $server_name == "Same than the domain" ]; then
    server_name=$domain
fi

Reply:
I tried what you suggested. It worked for me.